comet_core/api_helper.py

```diff
@@ -69,3 +70,29 @@ def decorated(*args, **kwargs):
         return f(*args, **kwargs)
 
     return decorated
+
+
+def valid_token(fingerprint, token):
+    """
+    Check if the token given in the request is valid by comparing
+    to the calculated API token.
+
+    Args:
+        fingerprint (str): the fingerprint to compute the API token with
+        token (str): the token to validate
+
+    Return:
+        True (bool): if the token is valid
+
+    Raises:
+        ValueError: if the token is not valid
+    """
+    hmac_secret = current_app.config.get('hmac_secret')
+    current_digest = hmac.new(bytes(hmac_secret, 'utf-8'),
+                              msg=bytes(fingerprint, 'utf-8'),
+                              digestmod=hashlib.sha256).hexdigest()
+    if not hmac.compare_digest(bytes(current_digest, 'utf-8'),
```

Comment:
I think we should add a return false at the bottom of this just to make sure if we change how we handle errors in the future that this would still return false.

Reply:
actually I dont think we should do this because its more confusing since we dont check the value